spec/integration_spec.rb

```diff
@@ -80,6 +87,44 @@
     end
   end
 
+  context "using plain tcp connection" do
+    it "should successfully connect to the server if ssl explicitely disabled" do
+      expect {
+        Lumberjack::Client.new(:port => tcp_port, :host => host, :addresses => host, :ssl => false)
+      }.not_to raise_error
+    end
+
+    it "should fail to connect to the server if ssl not explicitely disabled" do
+      expect {
+        Lumberjack::Client.new(:port => tcp_port, :host => host, :addresses => host)
+      }.to raise_error(RuntimeError, /Must set a ssl certificate/)
+    end
```

Comment:
We should have test for a **client** with SSL **on** connecting to a Server with SSL **off**.